### PR TITLE
virtio_fs_share_data: add error messages for pjdfstest and xfstest

### PR DESCRIPTION
**Pull request by lkotek**
PR adds error messages for pjdfstest and xfstest in virtio_fs_share_data test case.

ID: 1999727

Signed-off-by: Lukas Kotek <lkotek@redhat.com>

### CODE REVIEW

File: qemu/tests/virtio_fs_share_data.py
Comment (lkotek) [reply qemu/tests/virtio_fs_share_data.py]: Thanks for advice @PaulYuuu, you're right about it.  I changed it like you proposed.

File: qemu/tests/virtio_fs_share_data.py
Comment (lkotek) [reply qemu/tests/virtio_fs_share_data.py]: (Please see comment above.)

**Comment by lkotek** [comment]
This patch is tested and works as expected.

**Comment by PaulYuuu** [comment]
Hello @lkotek , you missed your `sign-off` in the commit. The format is like the following:
```
title           <- Limited to 72 characters. No period.
                <- Blank line
message         <- Any number of lines, limited to 72 characters per line.
                <- Blank line
Signed-off-by:  <- Signature and acknowledgment of licensing terms when
                   contributing to the project (created by git commit -s)
```

**Comment by lkotek** [comment]
> Hello @lkotek , you missed your `sign-off` in the commit. The format is like the following:
> 
> ```
> title           <- Limited to 72 characters. No period.
>                 <- Blank line
> message         <- Any number of lines, limited to 72 characters per line.
>                 <- Blank line
> Signed-off-by:  <- Signature and acknowledgment of licensing terms when
>                    contributing to the project (created by git commit -s)
> ```

@PaulYuuu Thanks for reminder. I changed it.

File: qemu/tests/virtio_fs_share_data.py
Comment (PaulYuuu) [inline qemu/tests/virtio_fs_share_data.py:295]: Hello @lkotek, it's a good habit to form a good code style, for this line, please limit the maximum length to 80 characters. Refer to: https://google.github.io/styleguide/pyguide.html#32-line-length

```suggestion
                s, o = session.cmd_status_output(cmd_pjdfstest % fs_dest,
                                                 io_timeout)
```
Or:
```suggestion
                status, output = session.cmd_status_output(
                    cmd_pjdfstest % fs_dest, io_timeout)
```

**Comment by PaulYuuu** [comment]
Hello @zhenyzha, For this patch, error messages will not be printed in the UI directly when the test failed. Could you help to review as well?

File: qemu/tests/virtio_fs_share_data.py
Comment (lkotek) [reply qemu/tests/virtio_fs_share_data.py]: Thanks I fully agree on that.  I committed your suggestion. Actually I wonder why `inspektor` didn't catch this before (I guess it was probably because one of the disabled PEPs).

**Comment by zhenyzha** [comment]
git am 2984.patch 
Applying: virtio_fs_share_data: add error messages for pjdfstest and xfstest

` (1/1) Host_RHEL.m8.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.5.0.aarch64.io-github-autotest-qemu.virtio_fs_share_data.run_stress.with_xfstest.with_local_source.with_cache.auto.arm64-pci: ERROR: Failed to build xfstests-dev (255.92 s)`

Thank you very much, now we can get the specific error message.
LGTM.Acked-by: Zhenyu Zhang <zhenyzha@redhat.com>

**Review by zhenyzha — APPROVED** [review]

**Comment by lkotek** [comment]
@zhenyzha Great, thanks a lot!

@CongLi May I ask you for review, please? This is my first-time contribution here, so I need also an approval from maintainer.

File: qemu/tests/virtio_fs_share_data.py
Comment (PaulYuuu) [inline qemu/tests/virtio_fs_share_data.py:298]:
```suggestion
                    logging.info(output)
```

**Comment by PaulYuuu** [comment]
Others LGTM.

**Review by PaulYuuu — APPROVED** [review]

**Pull request merged**
